src/man/sssd-ldap.5.xml

```diff
@@ -161,6 +161,45 @@
                     </listitem>
                 </varlistentry>
 
+                <varlistentry>
+                    <term>ldap_read_rootdse (string)</term>
+                    <listitem>
+                        <para>
+                            SSSD reads RootDSE to get information about LDAP and
+                            its capabilities. By default, this is done
+                            anonymously. However, this may not be permitted by
+                            the LDAP server. In such cases we can use this option
+                            to influence SSSD behavior.
+                        </para>
+                        <para>
+                            Allowed values are:
+                            <itemizedlist>
+                                <listitem>
+                                    <para>
+                                        anonymous
+                                    </para>
+                                </listitem>
+                                <listitem>
+                                    <para>
+                                        authenticated
+                                    </para>
+                                </listitem>
+                                <listitem>
+                                    <para>
+                                        never
+                                    </para>
+                                </listitem>
+                            </itemizedlist>
+                            By default, using the "anonymous" option, SSSD tries
```

Comment:
This line is redundant. 

```
By default, SSSD tries to read the RootDSE anonymously. In failure, SSSD will then try and authenticate the request. 
```

Reply:
@danlavu,

I do not see why is is redundant. (This formulation is @justin-stephenson 's improvemnt of my poor English :-) )

Can you elaborate?

Reply:
I'm sorry, redundant isn't the right word, obvious.

```
By default, using the "anonymous" option, SSSD tries to read RootDSE anonymously.
```

Why would it do anything else?

Reply:
I'm sorry, redundant isn't the right word, obvious.

```
By default, using the "anonymous" option, SSSD tries to read RootDSE anonymously.
```

Why would it do anything else?  =D

Reply:
Is this better?

`When SSSD tries to read the RootDSE anonymously (default behavior) and a failure occurs, SSSD will then try and authenticate the request.`

Reply:
That would be confusing to me even more.

The flow is `<bind (anonymous)> <read rootdse> -> <authenticate> -> <read rootdse if first attempt failed> -> <do authenticated request (search/update...)>`.

by request I would understand the last one but that has nothing to do with this option.

`...then try and authenticate the RootDSE request` ?

Reply:
Okay disregard my latest comment. I still don't see the issue with

`By default, SSSD tries to read the RootDSE anonymously. On failure, SSSD will then try and authenticate the request. 
`

Reply:
+1 
```
By default, SSSD tries to read the RootDSE anonymously. On failure, SSSD will then try and authenticate the request.
```

Reply:
+1 
```
By default, SSSD tries to read the RootDSE anonymously. On failure, SSSD will then try and authenticate the request.
```